src/components/Main/Modal/QRCodeModal/QRCodeModal.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <modal-frame title="QR Code" icon-mdi icon-name="qrcode">
+    <div
+      v-if="!err"
+      :class="$style.qrCodeModalImage"
+      :style="styles.container"
+    />
+    <modal-section v-else title="エラー" :description="err" />
+  </modal-frame>
+</template>
+
+<script lang="ts">
+import {
+  defineComponent,
+  reactive,
+  onMounted,
+  onUnmounted,
+  toRefs,
+  Ref
+} from '@vue/composition-api'
+
+import { makeStyles } from '@/lib/styles'
+import ModalFrame from '../Common/ModalFrame.vue'
+import ModalSection from '../Common/ModalSection.vue'
+import apis from '@/lib/api'
+
+const useStyles = (url: Ref<string>) =>
+  reactive({
+    container: makeStyles(() => ({
+      backgroundImage: `url(${url.value})`
+    }))
+  })
+
+const useQrCode = () => {
+  const state = reactive({
+    url: '',
+    err: ''
+  })
+
+  onMounted(() => {
+    makeBlob()
+      .then(value => {
+        state.url = value
+      })
+      .catch(err => {
+        state.err = err
+      })
+  })
+
+  onUnmounted(() => {
+    URL.revokeObjectURL(state.url)
+  })
+
+  const makeBlob = async () => {
+    const response = await apis
+      .getMyQRCode(false, {
+        responseType: 'arraybuffer'
+      })
+      .catch(e => {
+        throw e
+      })
+
+    const blob = new Blob([response.data], { type: 'image/png' })
+
+    return URL.createObjectURL(blob)
```

Comment:
```suggestion
    const { data } = await apis
      .getMyQRCode(false, {
        responseType: 'blob'
      })

    return URL.createObjectURL(data)
```

```js
  // `responseType` indicates the type of data that the server will respond with
  // options are: 'arraybuffer', 'document', 'json', 'text', 'stream'
  //   browser only: 'blob'
```
axiosのドキュメントにブラウザならblob返してくれるってあったからそれを使うと短くなるのと
async関数内でエラーが起きたときは勝手にthrowされるからcatchしてthrowしなくても同じになるよ

あと、この関数はblobをつくってるんじゃなくてblobに対応するobjectUrlを生成してるから関数名を変えたほうがいいと思う